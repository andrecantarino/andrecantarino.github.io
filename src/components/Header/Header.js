import React from 'react'
import styled from 'styled-components'

const StyledHeader = styled.header`
  display: flex;
  flex-direction: row;
  width: 100%;
  height: 7vh;
`

const Header = () => {
  return (
    <StyledHeader>
    </StyledHeader>
  )
}

export default Header;