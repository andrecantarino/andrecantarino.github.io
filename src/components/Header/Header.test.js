import React from 'react'
import { render } from '@testing-library/react'
import Header from './Header'

describe('Header | snapshot', () => {
  it('should take a snapshot', () => {
    const { asFragment } = render(<Header />)

    expect(asFragment(<Header />)).toMatchSnapshot()
  })
});