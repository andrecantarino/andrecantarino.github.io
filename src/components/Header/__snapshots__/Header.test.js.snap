// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header | snapshot should take a snapshot 1`] = `
<DocumentFragment>
  <header
    class="sc-bczRLJ fPvdEA"
  />
</DocumentFragment>
`;
